va/tlsalpn_test.go

```diff
@@ -446,7 +477,17 @@ func TestValidateTLSALPN01UnawareSrv(t *testing.T) {
 // will result in a problem with the invalid UTF-8 replaced.
 func TestValidateTLSALPN01BadUTFSrv(t *testing.T) {
 	chall := tlsalpnChallenge()
-	hs := tlsalpn01Srv(t, chall, IdPeAcmeIdentifier, 0, "localhost", "\xf0\x28\x8c\xbc")
+	hs, err := tlsalpn01Srv(t, chall, IdPeAcmeIdentifier, 0, "localhost", "\xf0\x28\x8c\xbc")
+	// In go1.16 and greater tlsalpn01Srv is expected to fail because of
+	// invalid unicode attempted in the certificate creation. If that
+	// error occurs, then the standard library has done it's job and this
+	// test is satisfied. If the error is for any other reason, the unit
+	// test will fail. In 1.15.x this error is not expected and the other
+	// test cases will continue.
+	if err != nil {
+		test.AssertContains(t, err.Error(), "cannot be encoded as an IA5String")
+		return
+	}
```

Comment:
This should also have a TODO and issue number to clean up once we're fully on Go 1.16.